### ISSUE TITLE
[Bug]: Split transactions' splits double-counted in Selected Balance

### ISSUE THREAD

**Issue by Ditti4** [actualbudget/actual#1732]
### Verified issue does not already exist?

- [X] I have searched and found no existing issue

### What happened?

When selecting split transactions and their individual splits, the "Selected Balance" value seems to double-count the last selected split:
![image](https://github.com/actualbudget/actual/assets/1223152/69b860a5-8462-43aa-ab03-294b850709ed)

This is especially confusing when range-selecting using Shift, as then all splits are double-counted. Here I first selected the main split transactinon, then I shift-selected the not-split transaction:
![image](https://github.com/actualbudget/actual/assets/1223152/68e80e12-c147-448b-8cde-d235e64de581)

I'm running version 2023.9.0 of both the client and the server.

### Where are you hosting Actual?

Docker

### What browsers are you seeing the problem on?

Chrome

### Operating System

Linux

**Comment by Kidglove57** [comment]
Yes indeed - and this is a long standing issue which means that the results from creating (and saving) filters (as reports) give incorrect totals.

**Comment by youngcw** [comment]
I believe this was fixed by #1751. Closing
